one-ui/src/main/java/com/marklogic/hub/oneui/auth/MarkLogicAuthenticationManager.java

```diff
@@ -107,10 +109,13 @@ public Authentication authenticate(Authentication authentication) throws Authent
         boolean stagingServerAccessible = false;
         try {
             stagingServerAccessible = dataServicesClient.checkConnection().isConnected();
-        } catch (Exception e) {
+        } catch (Exception ignored) {
+        }
+        if (!(hasManagePrivileges || stagingServerAccessible)) {
```

Comment:
we do not need to check stagingServerAccessible because it is a flag to indicate connection status.